Created Team Pics component for Home-page

trumanjchan opened the pull request:
The code is not responsive, so I will / we will have to change it later on.

## Proposed changes

We don't have a Team Pics component yet.
6/3/2020 Edit:
Members need to upload their photos into our Google Drive. Then their photo placeholders in the tsx file needs to be changed.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/UCMercedACM/Chapter-Website/blob/master/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Description

Made Responsive w/Queries on 6/3/2020. I followed this guide, https://www.codesmite.com/article/how-to-create-pure-css-hexagonal-grids, to create the hexagon honeycomb design. I also included media queries for the component's Header, and lots of comments in the scss file. What still needs to be changed are the placeholders in the tsx file. HackMerced members need to upload their photo onto our Google Drive (HackMerced>Website>the HUB>Team Pictures). In the tsx file, the src={...} will then need to be changed to their NAME.

trumanjchan committed:
Created Team Pics component for Home-page

The code is not responsive, so I will / we will have to change it later on.

maryharebear reviewed (changes requested):
Need to be responsive and use the Component className from your tsx file in your scss so its your own style and it wont affect others.

trumanjchan committed:
Made the TeamPics Component Responsive!

I used a different source online that had code for a responsive hexagon design, and edited it. I also added media queries for the Header of the component, "Our Team", so the font-size changes based on a specified resolution's width. We also decided not to use the linkedIn photos because the default resolution on those are too small, and would need to be scaled. For now, there are placeholders because members need to upload their photos into our Google Drive.
